src/python/pants/build_graph/build_configuration.py

```diff
@@ -180,6 +185,10 @@ def union_rules(self):
     """
     return self._union_rules
 
+  def target_adaptors(self):
+    """???"""
```

Comment:
I don't understand this doc comment.

Reply:
That's a TODO mark. I had intended to remove it before posting :)